docs/devupdate.md

```diff
@@ -9,3 +9,9 @@ tags:
 import { CodeBlock, CodeExample, InvalidExample, ValidExample } from '@site/src/components';
 
 This page highlights the important changes that are coming in Moodle 4.3 for developers.
+
+## Functions for generating and displaying user details
```

Comment:
A more appropriate heading would be something like "User-related API updates", then followed by a paragraph that describes the update. e.g. "New functions for generating and displaying user details have been created:", then followed by the list of the new user functions.